packages/table/src/headers/columnHeader.tsx

```diff
@@ -60,6 +65,12 @@ export interface IColumnHeaderProps extends ISelectableProps, IColumnIndices, IC
     viewportRect: Rect;
 
     /**
+     * Enables/disables the reordering interaction.
+     * @default true
+     */
+    isReorderable?: boolean;
```

Comment:
I agree with @themadcreator, this should not be exposed at this level if we don't expect to support it. How about if you make it `@internal`, does that work? If not, you could try to use component `context` to allow `<Table>` to talk to its sub components.